src/test/javascript/spec/component/shared/metis/postings-reactions-bar/answer-post-reactions-bar/answer-post-reactions-bar.component.spec.ts

```diff
@@ -0,0 +1,95 @@
+import * as chai from 'chai';
+import * as sinonChai from 'sinon-chai';
+import { ComponentFixture, getTestBed, TestBed } from '@angular/core/testing';
+import { MetisService } from 'app/shared/metis/metis.service';
+import { DebugElement, NO_ERRORS_SCHEMA } from '@angular/core';
+import * as sinon from 'sinon';
+import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
+import { MockPipe } from 'ng-mocks';
+import { OverlayModule } from '@angular/cdk/overlay';
+import { Reaction } from 'app/entities/metis/reaction.model';
+import { User } from 'app/core/user/user.model';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ReactionService } from 'app/shared/metis/reaction.service';
+import { MockReactionService } from '../../../../../helpers/mocks/service/mock-reaction.service';
+import { AccountService } from 'app/core/auth/account.service';
+import { MockAccountService } from '../../../../../helpers/mocks/service/mock-account.service';
+import { AnswerPostReactionsBarComponent } from 'app/shared/metis/postings-reactions-bar/answer-post-reactions-bar/answer-post-reactions-bar.component';
+import { AnswerPost } from 'app/entities/metis/answer-post.model';
+
+chai.use(sinonChai);
+const expect = chai.expect;
+
+describe('AnswerPostReactionsBarComponent', () => {
+    let component: AnswerPostReactionsBarComponent;
+    let injector: TestBed;
+    let fixture: ComponentFixture<AnswerPostReactionsBarComponent>;
+    let metisService: MetisService;
+    let answerPost: AnswerPost;
+    let reactionToCreate: Reaction;
+    let reactionToDelete: Reaction;
+
+    const user = { id: 1, name: 'username', login: 'login' } as User;
+
+    beforeEach(() => {
+        return TestBed.configureTestingModule({
+            imports: [HttpClientTestingModule, OverlayModule],
+            providers: [
+                { provide: MetisService, useClass: MetisService },
+                { provide: ReactionService, useClass: MockReactionService },
+                { provide: AccountService, useClass: MockAccountService },
+            ],
+            declarations: [AnswerPostReactionsBarComponent, MockPipe(ArtemisTranslatePipe)],
+            schemas: [NO_ERRORS_SCHEMA],
```

Comment:
Is there a specific reason we are ignoring errors in the template?

Reply:
It was due to `ngx-emoji` used in the template. I did not want to import the respective `EmojiModule`. But I changed it now in `post-reactions-bar.component.spec.ts` and `answer-post-reactions-bar.component.spec.ts` to:
````
...
imports: [HttpClientTestingModule, OverlayModule, EmojiModule],
schemas: [CUSTOM_ELEMENTS_SCHEMA],
...
````

Tests still run fast, so I think this is a reasonable change. Thanks for bringing that up.